Documentation/ceph-csi-drivers.md

```diff
@@ -104,7 +104,9 @@ statefulset.apps/csi-cephfsplugin-provisioner   1/1     4m5s
 statefulset.apps/csi-rbdplugin-provisioner      1/1     4m5s
 ```
 
-Once the plugin is successfully deployed, test it by running the following example.
+#### Liveness Sidecar
+All CSI plugin pods are deployed with a sidecar container that provides a prometheus metric for tracking if the CSI plugin is alive and runnning.
+
```

Comment:
please update the document on how to access the liveness endpoint and also add sample output